yarn-project/end-to-end/src/e2e_cli.test.ts

```diff
@@ -6,27 +6,18 @@ import { setup as e2eSetup } from './fixtures/utils.js';
 import { cliTestSuite } from './shared/cli.js';
 
 const HTTP_PORT = 9009;
-let RPC_URL = `http://localhost:${HTTP_PORT}`;
 const debug = createDebugLogger('aztec:e2e_cli');
 
 let http: ReturnType<typeof startHttpRpcServer>;
 let pxe: PXE;
 let teardown: () => Promise<void>;
 
-// Use Sandbox PXE URL if we're running against sandbox
-const { PXE_URL } = process.env;
-if (PXE_URL) {
-  RPC_URL = PXE_URL;
-}
```

Comment:
Why remove this? I think it's useful to be able to run this suite against a running server, so we don't have to recreate it every time. And it's not like it assumes it's a sandbox: it just requires a pxe.

Reply:
Yes, I realise the issues with the changes I made. Should be sorted now.